addons/hr_expense/models/hr_expense_sheet.py

```diff
@@ -717,11 +720,42 @@ def _do_reverse_moves(self):
         )
         draft_moves.unlink()
 
+    def _compute_default_accounting_date(self, most_recent_expense):
```

Comment:
You're right it should be named get_default_invoice_date but if we call it like this maybe it may be misleading because it may be meaning it is retrieving an existing invoice date maybe _calculate_invoice_date maybe better (it is no compute anymore this way) ?